docs/api/saved-objects/import.asciidoc

```diff
@@ -23,14 +23,14 @@ experimental[] Create sets of {kib} saved objects from a file created by the exp
 ==== Query parameters
 
 `createNewCopies`::
-  (Optional, boolean) Creates copies of saved objects, regenerates each object ID, and resets the origin. When used, potential conflict
-  errors are avoided.
+  (Optional, boolean) Creates new copies of saved objects, regenerates each object ID, and resets the origin. When used, potential conflict
+  errors are avoided. The default value is `true`.
 +
 NOTE: This cannot be used with the `overwrite` option.
 
 `overwrite`::
   (Optional, boolean) Overwrites saved objects when they already exist. When used, potential conflict errors are automatically resolved by
-  overwriting the destination object.
+  overwriting the destination object. The default value is `false`.
```

Comment:
I understand the motivation, swapping the default will allow us to collect more accurate usage data. But, this is a breaking change which would have a big impact on users who have automated some kind of syncing system. Although we don't have data we have strong anecdotal evidence that this is a widely used API. I'll defer to @kobelb but I don't feel like we have sufficient reason to break this API in a minor.

Reply:
Great points and it's definitely worth reconsidering.

I spoke to @kobelb about our motivation for this change and our long-term goals for both the Import and Copy workflows.

I think for the purposes of determining how many people are relying on the existing behavior for "pseudo sharing", it's more important to change the default for the Copy API -- we'll get more valuable information from that change.

It's less important to change the default for the Import API, though. And, long-term, we probably want import to be 1. space-aware, and 2. still have the option of detecting "exact match" conflicts. That would still allow us to get rid of all the `originId` nonsense when we get eventually get rid of the check-for-conflicts option on the Copy API.

TL;DR I am going to:
* Undo the changes for Import default options
* Add additional usage stats so we can see how many people are using these APIs via the Kibana UI or not